app/models/service_template_provision_task.rb

```diff
@@ -201,9 +201,13 @@ def after_ae_delivery(ae_result)
   def update_and_notify_parent(*args)
     prev_state = state
     super
+    return task_activated if state == "active" && prev_state != "active"
     task_finished if state == "finished" && prev_state != "finished"
```

Comment:
What do you think about refactoring this to be more generic:
```ruby
self.try("task_#{state}") if state != prev_state
```

The existing `task_finished` line on 205 can be deleted, `task_activated` would be renamed `task_active` and would not be required in the base model.